charts/kube-prometheus-stack/README.md

```diff
@@ -83,6 +83,9 @@ _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documen
 
 A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an incompatible breaking change needing manual actions.
 
+### From 15.x to 16.x
+Version 16 upgrades kube-state-metrics to v2.0.0. This includes changed command-line arguments and removed metrics, see this [blog post](https://kubernetes.io/blog/2021/04/13/kube-state-metrics-v-2-0/).
+
```

Comment:
BTW, list of CRDs to update below is missing link to PrometheusRules CRD.

Reply:
This should be okay, it hasn't changed in 6 months. In general I would assume we only support jumping one major release a time.

Reply:
Make sense, thanks for explanation! I found it a bit confusing.